packages/next/server/on-demand-entry-handler.ts

```diff
@@ -192,6 +195,9 @@ export default function onDemandEntryHandler(
           }
         }
 
+        // TODO: Move out of hot-reloader into a separate process
+        if (eslint) verifyEslintSetup(process.cwd(), pagesDir, pagePath)
```

Comment:
Are we wanting to update this in a follow-up PR since we're starting with dev linting disabled by default?

Reply:
Yep exactly. Will enable it by default in a future PR when it doesn't affect page compilation time + fast refresh